src/angular/icons/icon-settings.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, Input} from "@angular/core";
+import { Icons } from '../../iconsMap';
+import {Colors, Positions} from "./icons.component";
+
+
+@Component({
+    selector:'sdc-icon-settings',
+    templateUrl:'./icons-settings.template.html',
+    styles:['.row-settings{' +
+    'display:flex' +
+    '}' +
+    '.item-settings{' +
+    'display:flex;' +
+    'flex-direction:column;' +
+    'margin-right:20px;' +
+
+    '}']
+})
+
+
+export class IconSettingsComponent{
```

Comment:
This component is only used for the demo. Move it under `stories/ng2-component-lab/components`, and import it in `stories/ng2-component-lab/components.module.ts`.